### ISSUE TITLE
[Dashboard] Can't do multiple select on search page

### ISSUE THREAD

**Issue by jsanz** [CartoDB/cartodb#14822]
### Context

Issue found doing regular product usage, without any particular related project or third-party.

### Steps to Reproduce

1. Use the free text search
2. Try to select multiple items, for example to remove them

### Current Result

The mulitple select is not available.

### Expected result

I'd like to be able to perform the same multiple select available at the regular map or data pages.

### Browser and version

* Google Chrome Version 73.0.3683.103 (Official Build) (64-bit) - Linux

**Comment by jsanz** [comment]
@VictorVelarde since this is not urgent, not sure if this is RT material or something to track elsewhere.

**Comment by VictorVelarde** [comment]
Please have a quick look at this @jesusbotella, to see if this can be also tackled by @alejandraarri at RT, in the context of https://github.com/CartoDB/product/issues/243 (also dashboard & search related)

**Comment by jesusbotella** [comment]
Not related to https://github.com/CartoDB/product/issues/243.

The multiple selection is not enabled on purpose within Search page, because of the presence of maps and datasets and the mess of selecting both types of visualization with different actions.

If we want multiple selection, we'd need to think about it from a UX/Design POV.

**Comment by VictorVelarde** [comment]
So then I'm pinging @alonsogarciapablo for awareness on this and a proper discussion

**Comment by alonsogarciapablo** [comment]
Yep, this is a feature not a bug. cc: @kukukaka 

@jsanz when you searched, did you have any specific task in mind (eg: I'm going to delete all maps with end with "copy")?

**Comment by jsanz** [comment]
> @jsanz when you searched, did you have any specific task in mind (eg: I'm going to delete all maps with end with "copy")?

Correct, I wanted to delete quite a few datasets, that involves several clicks for each so the multiple select from the main Data page is convenient

**Comment by alonsogarciapablo** [comment]
Thank you @jsanz. We'll keep this open to keep track of more requests like yours.

cc: @kukukaka
